Clean Swift/Clean Swift/SpaceModule/SpaceProtocols.swift

```diff
@@ -0,0 +1,35 @@
+// SpaceProtocols.swift
+// Copyright © Alexandr T. All rights reserved.
+
+import UIKit
+
+/// Протокол для хранения данных экрана космоса
+protocol SpaceDataStore {
+    var space: Astronomy { get set }
+}
+
+/// Протокол вьюшки с информацией о космосе
+protocol SpaceDisplayLogicProtocol {
+    func fetchedOrders(viewModel: SpaceList.FetchAstronomyList.ViewModel)
```

Comment:
```suggestion
    func displayInitForm(viewModel: SpaceList.FetchAstronomyList.ViewModel)
```
посмотри видео из важного Сашино на 9:50